src/rolodex/rolodexPrinter.js

```diff
@@ -1,11 +1,10 @@
 import people from './people.json';
 
-people.forEach(function (person) {
-  const names = person.name.split(' ');
-  const firstName = names[0];
-  const lastName = names[1];
-  const email = person.email;
-  const phone = person.phone;
+people.forEach(([name, email, phone] = [people.name, people.email, people.phone]) => {
+  const { firstName, lastName } = name.split(' ');
```

Comment:
This will not work, `name.split` returns an array.
`const [firstName, lastName] = name.split(' ')

Reply:
ah..  dang.  I was judging success by 'npm test'.  I don't think I actually tried to run any of them individually.  😳